### DISCUSSION TITLE
Option to add an icon into inputSwitch

### DISCUSSION THREAD

**Discussion by stephanebouget** [primefaces/community#1020] [PrimeNG] [answered]
It would be nice to have an option to integrate icons into the component

For instance:
trueIcon    string    pi-sun
falseIcon   string    pi-moon

Would give:
![image](https://github.com/primefaces/community/assets/13203455/ff0368e9-8453-4a39-acc5-b953e40e7629)
![image](https://github.com/primefaces/community/assets/13203455/d1fe828a-0294-451d-a49f-40acd5893682)

**Comment by cetincakiroglu** [comment, marked as answer]
Hi,

Thanks for the feedback and the feature suggestion. We'll wait for the community interaction to develop this feature.
